static/src/javascripts/projects/commercial/modules/third-party-tags.spec.js

```diff
@@ -84,8 +90,18 @@ describe('third party tags', () => {
         };
         it('should add a script to the document', () => {
             _.reset();
+            onIabConsentNotification.mockImplementation(callback =>
+                callback({
+                    '1': true,
+                    '2': true,
+                    '3': true,
+                    '4': true,
+                    '5': true,
+                })
+            );
+            console.log('*** before: ', document.scripts);
             insertScripts([fakeThirdPartyTag]);
-
+            console.log('*** after: ', document.scripts);
```

Comment:
There's a  couple of stray console logs left in here that should be removed.

Reply:
Fixed.